test/integration/default/serverspec/windows_spec.rb

```diff
@@ -1,26 +0,0 @@
-# these tests only for windows
-require 'spec_helper'
-
-expected_rules = [
```

Comment:
> And you seemed to happy with this on your comment on Feb 11. What was changed since then? 

You rebased against master, so I am now able to fully review what the real changes are.

Separating test suites != deleting tests. I'd like to understand why we are removing tests, instead of just adding additional tests.